modules/etargetBidAdapter.js

```diff
@@ -57,18 +57,44 @@ export const spec = {
       data: bidderRequest,
       bids: validBidRequests,
       netRevenue: netRevenue,
+      metaData: getMetaData(),
       bidder: 'etarget',
       gdpr: gdprObject
     };
-
+    function getMetaData() {
```

Comment:
Moving the request here,  please also gather this info from the ortb2 object

Reply:
problem is that ortb2 is available only when portal will fill info into this object, what we want to achieve is to gather info every time our adapter will be used. Is there any leagal/technical problem with our implementation? Why we cant collect info by ourselfs?

Reply:
> Please also fix #6650 ; it's being enforced imminently

I started working on this feature but i need to create new fork because when i edit things on old fork then circleci have always problems with somethink. But i cant create new fork before this pull requerst is closed. So can you please merge this new feature if there is no real reason why not to merge? THENKS.